gb-backend/src/main/groovy/nl/thehyve/gb/backend/representation/DimensionPropertiesRepresentation.groovy

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright (c) 2019  The Hyve B.V.
+ *  This file is distributed under the GNU Affero General Public License
+ *  (see accompanying file LICENSE).
+ */
+
+package nl.thehyve.gb.backend.representation
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import groovy.transform.Canonical
+import groovy.transform.CompileStatic
+
+@Canonical
+@CompileStatic
+class DimensionPropertiesRepresentation {
+
+    /**
+     * Dimension name
+     */
+    String name
+
+    /**
+     * If true, this dimension will be inlined in the cell. Only present if true.
+     */
+    Boolean inline
```

Comment:
I suggest having only fields that gbe cares at the moment: Is it `name` only? jackson wouldn't mind, right?